platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -432,6 +432,18 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark Selecting Annotations
 
+/**
+ Returns a Boolean value indicating whether the annotation can be selected.
+ 
+ If the return value is `YES`, or if this method is absent from the delegate,
+ the annotation will be selected.
```

Comment:
I don’t think it’s the case that, if this method is unimplemented, the annotation automatically gets selected when it’s added to the map view. It also doesn’t seem to affect whether the developer can select it programmatically via `-selectAnnotation:`. Rather, it seems to only affect whether an annotation can be selected _by a user gesture_:

> If the return value is `YES`, the user can select the annotation by tapping [clicking] on it. If the delegate does not implement this method, the default value is `YES`.